FWCore/Common/src/EventBase.cc

```diff
@@ -30,7 +30,7 @@ namespace {
       }
    };
    typedef tbb::concurrent_unordered_map<edm::ParameterSetID, edm::TriggerNames, key_hash> TriggerNamesMap;
-   [[cms::thread_safe]] static TriggerNamesMap triggerNamesMap;
+   [[cms::thread_safe]] TriggerNamesMap triggerNamesMap;
```

Comment:
The `[[cms::thread_safe]]` still needs to remain since this is still a 'globally' accessible object and therefore the static analysis will flag it.